app/presenters/publishing_api/contact_presenter.rb

```diff
@@ -71,34 +68,40 @@ def contact_type
     end
 
     def details
-      details = { description: description, contact_type: contact_type, title: title }
-      details[:contact_form_links] = [contact_form_links] if contact_form_url
-      details[:post_addresses] = [post_address] if postal_code
-      details[:email_addresses] = [email_address] if email
+      details = {
+        title: title,
```

Comment:
I think you changed the schema so these aren't required anymore - just wondering if you meant to remove these now?

Reply:
I'm not 100% sure what the consequences are of removing them, but am confident they shouldn't be used.